doc/EnvironmentVariables.md

```diff
@@ -75,7 +75,7 @@ You can set Edge Agent and Edge Hub environment variables in the Azure portal. I
 | MaxConnectedClients                       | Maximum number of downstream clients allowed to connect                                                                                            | int32                                                                 | 101 (100 clients + 1 EdgeHub) |
 | MaxUpstreamBatchSize                      | Max number of messages to concurrently send upstream                                                                                               | int32                                                                 | 10                            |
 | MessageAckTimeoutSecs                     | Time span to wait for sending a message downstream to a leaf device                                                                                | int32                                                                 | 30                            |
-| MessageCleanupIntervalSecs                | Time interval for task that cleans up messages in edgeHub's store                                                                                  | int32                                                                 | 1800                          |
+| MessageCleanupIntervalSecs                | This setting defines the time interval for a task that cleans up messages in EdgeHub's store. Note that messages are only removed from the store when this task runs. If you use a TTL (Time To Live) that is shorter than the default cleanup interval, please adjust the cleanup interval accordingly to ensure timely message removal.                                                                                  | int32                                                                 | 1800                          |
```

Comment:
When you talk about the TTL setting, are you referring to _timeToLiveSecs_ documented in https://github.com/Azure/iotedge/blob/main/doc/Route_priority_and_TTL.md? Do we want a link here so there's no doubt?

Reply:
sure, udpated